lambdas/service/app.py

```diff
@@ -870,69 +940,48 @@ def start_manifest_generation():
-    response is of status 200, only the `Status` field of the body will be 301. The intent is to emulate HTTP while
-    bypassing the default client behavior which, in most web browsers, is to ignore `Retry-After`. The response
-    described here is intended to be processed by client-side Javascript such that the recommended delay in
-    `Retry-After` can be handled in Javascript rather that relying on the native implementation by the web browser.
-
-    If the manifest generation is done and the manifest is ready to be downloaded, the response will be:
-
-    ```
-    {
-        "Status": 302,
-        "Location": "https://…"
-    }
-    ```
+# Copy of path_spec required due to FIXME: https://github.com/DataBiosphere/azul/issues/1646
+@app.route('/fetch/manifest/files', methods=['GET'], cors=True, path_spec=copy.copy(manifest_path_spec), method_spec={
+    'tags': ['Manifests'],
+    'summary': 'Request a download link to a manifest file and check status',
+    'description': format_description('''
+        Initiate a manifest generation or check the status of an already ongoing
+        generation, returning a 200 response with simulated HTTP headers in the
+        body.
+    '''),
+    'responses': {
+        '200': {
+            **{
+                'content': {
+                    'application/json': {
+                        'schema': schema.object(
+                            Status=int,
+                            Location={'type': 'string', 'format': 'url'},
+                            **{'Retry-After': schema.optional(int)}
+                        )
+                    }
+                }
+            },
+            'description': format_description('''
+                Manifest generation has been initiated or status checked of an
+                already ongoing generation, emulating the response code and
+                headers of the `/manifest/files` endpoint. Note that the actual
+                HTTP response will have status 200 while the `Status` field of
+                the body will be 301 or 302. The intent is to emulate HTTP while
+                bypassing the default client behavior, which (in most web
+                browsers) is to ignore `Retry-After`. The response described
+                here is intended to be processed by client-side Javascript such
+                that the recommended delay in `Retry-After` can be handled in
+                Javascript rather than relying on the native implementation by
+                the web browser.
 
-    The client should request the URL given in the `Location` field. The URL will point to a different service and
-    the client should expect a response containing the actual manifest. Currently the `Location` field of the final
-    response is a signed URL to an object in S3 but clients should not depend on that.
-    """
+                See `/manifest/files` [(link)](#operations-Manifests-get_manifest_files)
```

Comment:
```suggestion
                See [`/manifest/files`](#operations-Manifests-get_manifest_files) endpoint 
```

It works in GH Markdown:

[`some text`](https://github.com/DataBiosphere/azul/pull/1435)

Reply:
Also, you're making it a little easy for yourself here. I though we agreed on our last call that it needs to be made clear that the **fields** of a `/fetch/manifest/files` response body directly relate to the response **headers** from the `/manifest/files` endpoint. Let's try to nail this in the next review cycle. I'm getting tired of writing more text in review comments on documentation than there actually is documentation text.

Reply:
I've reworded the sentence with the link and the preceding paragraph in attempt to better highlight the relationship of the response fields to the response headers they emulate.